rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,591 @@
+			export GEODESIC_TRACE="${1#*=}"
+			;;
+		--no-custom*)
+			export GEODESIC_CUSTOMIZATION_DISABLED=true
+			;;
+		--no-motd*)
+			export GEODESIC_MOTD_ENABLED=false
+			;;
+		--workspace=*)
+			unset WORKSPACE_FOLDER_HOST_DIR
+			;& # fall through
+		--*)
```

Shellcheck (0.10.0)

[warning] This pattern always overrides a later one on line 144.

(SC2221)